keyboards/cipulot/60xt/info.json

```
@@ -0,0 +1,117 @@
+{
+    "manufacturer": "Cipulot",
+    "keyboard_name": "60XT",
+    "maintainer": "Cipulot",
+    "bootloader": "stm32-dfu",
+    "build": {
+        "lto": true
+    },
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
```

Generally preferred to be enabled if possible.
```suggestion
    "features": {
        "bootmagic": true,
```